### PR TITLE
Hannah/physics deletion site specific

### PR DESCRIPTION
**Pull request by hannah-whelan**

### CODE REVIEW

File: src/app/components/elements/PageTitle.tsx [hunk 1]
```diff
@@ -94,11 +91,7 @@ export const PageTitle = ({currentPageTitle, subTitle, disallowLaTeX, help, clas
             <meta property="og:title" content={currentPageTitle} />
         </Helmet>
         {audienceViews && <AudienceViewer audienceViews={audienceViews} />}
-        {help && !showModal && <React.Fragment>
```
Comment (jmgq) [inline src/app/components/elements/PageTitle.tsx:97]: This looks wrong... For CS, `showModal` would have been always false. So the help section would have always been visible (providing that `help` is truthy, and similarly the "link" section would have always been false for CS, but here we're now displaying it if there's a `modalId`.

File: src/IsaacApiTypes.tsx [hunk 2]
```diff
@@ -511,7 +511,7 @@ export type Stage = "year_7_and_8" | "year_9" | "gcse" | "a_level" | "further_a"
 
 export type ExamBoard = "aqa" | "cie" | "edexcel" | "eduqas" | "ocr" | "wjec" | "all";
 
-export type Difficulty = "practice_1" | "practice_2" | "practice_3" | "challenge_1" | "challenge_2" | "challenge_3";
```
Comment (RobertSaylesAND) [inline src/IsaacApiTypes.tsx:514]: Are these difficulties specifically associated with physics for some reason or has cs just not made use of them so far?

Comment (RobertSaylesAND) [inline src/IsaacApiTypes.tsx:514]: Are these difficulties specifically associated with physics for some reason or has cs just not made use of them so far?

File: src/app/components/content/IsaacQuestion.tsx [hunk 3]
```diff
@@ -62,20 +57,9 @@ export const IsaacQuestion = withRouter(({doc, location}: {doc: ApiTypes.Questio
         !!locked
     );
 
-    const tooManySigFigsFeedback = <p>
```
Comment (RobertSaylesAND) [inline src/app/components/content/IsaacQuestion.tsx:65]: Are significant figures irrelevant to CS? Aside from symbolic chemistry the supported question types should be the same, I would have thought?

File: src/app/components/elements/PageTitle.tsx (hunk 1 above)
Comment (hannah-whelan) [reply src/app/components/elements/PageTitle.tsx:97]: Not sure why I did this as it is clearly wrong!! Should be corrected now

File: src/IsaacApiTypes.tsx (hunk 2 above)
Comment (hannah-whelan) [reply src/IsaacApiTypes.tsx]: Dug into this some more and had a play about. I've reverted it to how it was before (and also for Constants.ts) to preserve for future functionality, and it's easy to get Practice 3 and Challenge 3 to correctly display questions in the Question Finder for CS, however the icons are not correct so I'm going to create a separate ticket for this one.  It looks like there's only 4 "practice_3" questions for CS and 13 "challenge_3" so it's impacting very little content compared to the other difficulty levels (hundreds of questions for each)

File: src/app/components/content/IsaacQuestion.tsx [hunk 4]
```diff
@@ -125,17 +109,15 @@ export const IsaacQuestion = withRouter(({doc, location}: {doc: ApiTypes.Questio
...
+               <IsaacLinkHints questionPartId={doc.id as string} hints={doc.hints} />
 
                 {/* Validation Response */}
                 {showQuestionFeedback && validationResponse && !canSubmit && <div className={`validation-response-panel p-3 mt-3 ${correct ? "correct" : ""}`}>
                     <div className="pb-1">
-                        <h1 className="m-0">{sigFigsError ? "Significant Figures" : correct ? "Correct!" : "Incorrect"}</h1>
+                        <h1 className="m-0">{correct ? "Correct!" : "Incorrect"}</h1>
                     </div>
                     {validationResponse.explanation && <div className="mb-2">
-                        {invalidFormatError ? invalidFormatFeeback : tooManySigFigsError ? tooManySigFigsFeedback : tooFewSigFigsError ? tooFewSigFigsFeedback :
```
Comment (jmgq) [inline src/app/components/content/IsaacQuestion.tsx:136]: I'm glad this aberration is gone. These chained ternary operators are very hard to read and should be avoided.

**Pull request closed without merging**